pkg/urbit/include/vere/meta.h

```diff
@@ -0,0 +1,18 @@
+//! @file meta.h
+
+#ifndef U3_VERE_META_H
+#define U3_VERE_META_H
+
+//==============================================================================
+// Types
+//==============================================================================
+
+//! Pier metadata managed by an event log.
+typedef struct {
+  c3_d who_d[2]; //!< identity
+  c3_o fak_o;    //!< fake bit
+  c3_w lif_w;    //!< lifecycle length
+  c3_w ver_w;    //!< event log version
```

Comment:
This is the event log version number, not the epoch version number. Each epoch also has a version number, although there is no field in the `u3_epoc` struct at the moment. See `epo_ver_w` in `epoc.c`. I assumed that it would be useful to have a version number on the event log too in case we, say, make interface changes to `saga`.

Reply:
Where is this number sourced from. Is there a version file in `log/` in addition to each `log/$epoch`?

Reply:
Bump.

Reply:
Sorry about the delay here. This thread got lost in the shuffle. This use of `ver_w` was a mistake on my part. I intended it to represent the event log version number but failed to persist it, so it was essentially useless. Because each epoch is versioned, I think we can get away without a version number on the event log itself, so I've removed it. What do you think?